src/Forms/SearchableDropdownField.php

```diff
@@ -15,7 +15,7 @@ class SearchableDropdownField extends DropdownField
 
     public function __construct(
         string $name,
-        string $title,
+        ?string $title,
```

Comment:
I've made $title nullable here and in in SearchableMultiDropdownField because it was failing CI - https://github.com/silverstripe/silverstripe-framework/actions/runs/7241075403/job/19724770031#step:12:118

$title has a default value of null in parent::__construct(), though I'd prefer not to give it a default value of null here otherwise it means we need to make $source nullable as well and we'd then need to throw an exception if a null value of $source is passed, which is just messy.

Reply:
Ahh, I should have caught this in the initial review.

As much as I agree that having `$source` have a default value isn't 100% ideal, it _is_ an expectation that form fields will have a default value of `null` for the title in their constructor. I don't think we have a single form field in core that doesn't follow this pattern currently - it's pretty common to just skip that argument.

It should be possible to do either of these:
```php
SearchableDropdownField::create('MyRelation', source: MyClass::get())
```
```php
SearchableDropdownField::create('MyRelation')->setSource(MyClass::get())
```

Reply:
Updated.

All the code in SearchableDropdownFieldTrait will check that $this->sourceList is truthy before doing anything with it, so it's safe to have a default value of null